examples/resourceServer/client/index.html

```diff
@@ -1,26 +1,51 @@
 <head>
-    <title>resourceServer</title>
+    <title>ResourceServer</title>
 </head>
 
 <body>
-    <h1>Welcome to resourceServer!</h1>
+    <h1>Welcome to ResourceServer!</h1>
 
     {{> authorize}}
 </body>
 
 <template name="authorize">
+    <p>
+      This <code>meteor-oauth2-server</code> package currently provides functionality for both the AuthenticationServer and the ResourceServer.  This functionality may be split into separate packages and/or examples in the future; however, for the time being, AuthenticationServer and ResourceServer are the same application.
+    </p>
     <p>The following steps represent the steps that are necessary to properly negotiate an oauth2 token. While these
-    steps work on this site, this site is for testing and demo purposes only. Do not use this configuration in production.</p>
-    <p>You'll know if everything is working if you can get to Step 6 and the user ids match.</p>
+    steps work on this site, this site is for testing and demo purposes only. Do not use this configuration in production.  You'll know if everything is working if you can get to Step 6 and the user ids match.</p>
     <hr />
 
-    <h2>Step 0 - Add an an authorized client</h2>
+    <h2>Step C1.1 - Generate Client ID & Secret</h2>
```

Comment:
Whichever.  :)

Lets take it out then, and if people ask for it to be put back in, then add it back.  It's not overly complex.  

I suppose I don't see the difference between a template project and an example.  People are going to refer to the example in building their own projects; so I tend towards streamlining that.